.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         - name: Security audit
           linux: bandit
 
-        - name: PEP 8
+        - name: Code style check
```

Comment:
Do you really need both flake8 and ruff style checks? Maybe it is time to get rid of flake8 checks here (and in tox.ini and .flake8 file)

Reply:
I removed ci check but why remove `.flake8` file?

Reply:
It is no needed anymore.